spec/calendar_spec.rb

```diff
@@ -51,7 +51,7 @@
         cal.add_rule('start_time', 'equals', '10:00')
 
         filtered_ical = cal.filtered_calendar
-        expect(filtered_ical).to eq(filtered_calendar)
+        expect(filtered_ical.gsub("\r\n", "\n")).to eq(filtered_calendar.gsub("\r\n", "\n"))
```

Comment:
You're performing the same transformation on both sides? What gives here?

Reply:
I can just say it works like this here in unix environment and on my windows. Is gsub making regex and thereby harmonizing line endings like that? Let me check...

Reply:
You can also have a look, made it proper regex to make it platform independent